app/views/application/assistancedetails/needsSupportOnlineAssessments.scala.html

```diff
@@ -5,8 +5,8 @@
 
 <section>
     <fieldset class="form-group">
-        <legend class="form-label">Will you need extra support for your online assessments?</legend>
-        <p class="form-hint">This includes the online tests, eTray and the video interview</p>
+        <legend class="form-label">Will you need extra support for your online tests?</legend>
+        <p class="form-hint">This includes the e-tray and the video interview</p>
```

Comment:
will check on this one, well reminded though as need to go through and check we've been consistent with whatever we're calling e-tray